package.json

```diff
@@ -2,12 +2,15 @@
   "name": "@bcgsc-pori/graphkb-parser",
   "version": "1.1.3",
```

Comment:
given the breaking changes should this be bumperd to 2?

Reply:
we don't bump version numbers except in release PRs b/c more generally things can be additive with other changes.